coalals/results.py

```diff
@@ -0,0 +1,67 @@
+from json import loads
+
+
+class Diagnostics:
+
+    @classmethod
+    def from_coala_json(cls, json_op):
+        coala_op = loads(json_op)['results']
+        warnings, fixes = [], []
+
+        def convert_offset(x):
+            return x - 1 if x else x
+
+        for section, coala_warnings in coala_op.items():
+            for warning in coala_warnings:
+                severity = 3 - warning['severity']
```

Comment:
It is better to add description here to illustrate why we are using `3 - warning['severity']` instead of `warning['severity']`.